daniel-robert/server.js

```diff
@@ -172,7 +174,7 @@ function loadArticles() {
 
 function loadDB() {
   // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
-  // Put your response here...
+  // Put your response here... 3, 4, 5. loadDB is interacting with the loadArticles function in article.js. This is the Create part of CRUD.
```

Comment:
Is loadDB being called in loadArticles function? It calls the loadArticle but the solution we are looking for is where loadDB is being called.

Reply:
I didn't say loadDB is being called by the loadArticles function, but its the function that it "interacts with"  loadDb isn't called by a function specifically.  It is called within the server.js file @ line 140.